### DISCUSSION TITLE
why can't  I get a right output of stable diffusion follow your official document?

### DISCUSSION THREAD

**Discussion by yufengyao-lingoace** [hpcaitech/ColossalAI#2454] [Community | Q&A]
why can't  I get a right output of stable diffusion follow your official document?

![WX20230111-183123@2x](https://user-images.githubusercontent.com/92853025/211794328-2090925a-e9a5-466b-8a6b-5f48a4e3443e.png)

The command I used:

python main.py --logdir /tmp/ -t -b configs/train_colossalai.yaml
python scripts/txt2img.py --prompt "a photograph of an astronaut riding a horse" --plms
    --outdir ./output \
    --config path/to/logdir/checkpoints/last.ckpt \
    --ckpt /path/to/logdir/configs/project.yaml  \

**Comment by binmakeswell** [comment]
https://github.com/hpcaitech/ColossalAI/discussions/2460
